utils/error_injection.hh

```diff
@@ -237,8 +256,8 @@ public:
         return true;
     }
 
-    void enable(const std::string_view& injection_name, bool one_shot = false) {
-        _enabled.emplace(injection_name, injection_data{one_shot});
+    void enable(const std::string_view& injection_name, bool one_shot = false, error_injection_parameters parameters = {}) {
```

Comment:
The corresponding `enable` function for release mode should be modified to match the signature. You're lucky it compiled -- apparently everything that uses `enable` is only compiled conditionally in non-release modes, like `enable_initial_error_injections` from main.cc.